setup.cfg

```diff
@@ -0,0 +1,75 @@
+[metadata]
+name = napari_animation
+version = 0.0.1
```

Comment:
if you're going to use `setuptools_scm` for version management (which is what is indicated by the `use_scm_version` bit in `setup.py`) ... then you can leave out the version here.  If we have it in the napari `setup.cfg`, it's for "odd" reasons :)